ownCloud/AppDelegate.swift

```diff
@@ -51,6 +51,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		OCExtensionManager.shared.addExtension(WebViewDisplayViewController.displayExtension)
 		OCExtensionManager.shared.addExtension(PDFViewerViewController.displayExtension)
 		OCExtensionManager.shared.addExtension(ImageDisplayViewController.displayExtension)
+		OCExtensionManager.shared.addExtension(OpenInAction.actionExtension)
```

Comment:
Only cosmetic: a space here would help to visually group the different kinds of extension.

Reply:
Made it!